pkg/jobmanager/start.go

```diff
@@ -75,9 +75,9 @@ func (jm *JobManager) startJob(ctx xcontext.Context, j *job.Job, resumeState *jo
 	jm.jobsMu.Lock()
 	defer jm.jobsMu.Unlock()
 	jobCtx, jobCancel := xcontext.WithCancel(ctx)
-	jobCtx2, jobPause := xcontext.WithNotify(jobCtx, xcontext.ErrPaused)
+	jobCtx, jobPause := xcontext.WithNotify(jobCtx, xcontext.ErrPaused)
 	jm.jobs[j.ID] = &jobInfo{job: j, pause: jobPause, cancel: jobCancel}
-	go jm.runJob(jobCtx2, j, resumeState)
+	go jm.runJob(jobCtx.WithField("job_id", j.ID), j, resumeState)
```

Comment:
this won't cover the resume case. why ont move it to runJob itself?

Reply:
"resume job" also calls `startJob`, so it will cover. But you anyway right (seems more stable to further changes to move to `runJob`), so moved to `runJob`.